data-sources/api/src/commonMain/kotlin/EpisodeSort.kt

```diff
@@ -129,3 +129,18 @@ fun EpisodeSort(int: BigNum): EpisodeSort {
     if (int.isNegative()) return Special(int.toString())
     return EpisodeSort(int.toString())
 }
+
+/**
+ * @see EpisodeType
+ */
+fun EpisodeSort(int: BigNum, type: EpisodeType): EpisodeSort {
```

Comment:
建议删除原来的 `fun EpisodeSort(int: BigNum): EpisodeSort {`, 改成用默认参数:
```suggestion
fun EpisodeSort(int: BigNum, type: EpisodeType = EpisodeType.MainStory): EpisodeSort {
```

让重载数量尽快少一点